### PR TITLE
Use black

### PR DESCRIPTION
**Pull request by schneiderfelipe**
This is so that we can take a look at how the repository will look like if we run [`black -l 100`](https://github.com/psf/black) over it. This PR would solve #1122. I also sorted import statements with [`isort`](https://github.com/PyCQA/isort) and rerun `black -l 100`. I separated it in three different commits so that we can see the differences.

A bird's eye view on the changes using [`cloc`](https://github.com/AlDanial/cloc):

**original**:

```
$ cloc .
...
--------------------------------------------------------------------------------
Language                      files          blank        comment           code
--------------------------------------------------------------------------------
...
Python                          271           7601          10591          25507
...
```

**this PR**:

```
$ cloc .
...
--------------------------------------------------------------------------------
Language                      files          blank        comment           code
--------------------------------------------------------------------------------
...
Python                          271           7662          10572          26166
...
```

EDIT: we could enforce or require some [pre-commit hook](https://pre-commit.com/) to maintain this in the future.

### CODE REVIEW

**Comment by schneiderfelipe** [comment]
`isort` was indeed the problem.

**Comment by berquist** [comment]
I've added a preliminary `pyproject.toml` for just black and isort (because we shouldn't use black without isort) and a plausible pre-commit config, though I haven't figured out how to get it to also exclude data files outside of `data` (with something like `\.(adfin|adfout|arc|dal|dat|com|cube|fchk|gjf|in|inp|log|nw|mop|out|txt)$ `).

File: cclib/parser/data.py [hunk 1]
```diff
@@ -105,87 +105,91 @@ class ccData:
...
+        "transprop": Attribute(dict, "electronic transitions", "transitions"),
+        "vibanharms": Attribute(numpy.ndarray, "anharmonicity constants", "vibrations"),
+        "vibdisps": Attribute(numpy.ndarray, "displacement", "vibrations"),
+        "vibfreqs": Attribute(numpy.ndarray, "frequencies", "vibrations"),
+        "vibfconsts": Attribute(numpy.ndarray, "force constants", "vibrations"),
+        "vibirs": Attribute(numpy.ndarray, "IR", "vibrations:intensities"),
+        "vibramans": Attribute(numpy.ndarray, "raman", "vibrations:intensities"),
+        "vibrmasses": Attribute(numpy.ndarray, "reduced masses", "vibrations"),
+        "vibsyms": Attribute(list, "vibration symmetry", "vibrations"),
+        "zpve": Attribute(float, "zero-point correction", "properties:energies"),
```
Comment (berquist) [inline cclib/parser/data.py:182]: Same comment about alignment here.

**Review by berquist — COMMENTED** [review]
I didn't look at all the changes, but of course most of them are positive and minor.  We will have to make decisions about some explicitly-aligned code.

**Comment by berquist** [comment]
Closing because this was just to see how things would change, and we're going to apply these changes once (most of) the PR queue is cleared out.

**Pull request closed without merging**